app/client/components/userNav.tsx

```diff
@@ -145,7 +145,7 @@ export class UserNav extends React.Component {
     },
     {
       title: "Digital Pack",
-      link: `${profileHostName}/digitalpack/edit`,
+      link: `/digitalpack`,
```

Comment:
Utterly minor point - we could change this back to double-quotes now that there's no string interpretation required.

Reply:
Done, along with all the others in this file.